static/js/compose_closed_ui.js

```diff
@@ -122,10 +168,16 @@ export function initialize() {
     // Click handlers for buttons in the compose box.
     $("body").on("click", ".compose_stream_button", () => {
         compose_actions.start("stream", {trigger: "new topic button"});
+        // When pm-with deactivated user compose box remains closed,
+        // however if a user clicks on new stream message or private message buttons,
+        // both the buttons should work as intended, thus initialize expand_compose_box function
+        // to open compose_box.
+        compose_actions.expand_compose_box();
     });
 
     $("body").on("click", ".compose_private_button", () => {
         compose_actions.start("private", {trigger: "new private message"});
+        compose_actions.expand_compose_box();
```

Comment:
You should check for the `trigger` inside `compose_actions.start` to verify if you should expand the compose box, instead of calling `expand_compose_box` in a different line.

Reply:
Sure will look into all the suggested changes